release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -141,6 +141,27 @@ Before you set a path value for the `template` parameter, ensure that the defaul
 
 {product-title} {product-version} is now supported on 64-bit ARM architecture-based Google Cloud Platform installer-provisioned and user-provisioned infrastructures. You can also now use the `oc mirror` CLI plug-in disconnected environments on 64-bit ARM clusters. For more information about instance availability and installation documentation, see xref:../installing/installing-preparing.adoc#supported-installation-methods-for-different-platforms[Supported installation methods for different platforms].
 
+[id="ocp-4-14-azure-custom-rhcos"]
+==== Using a custom {op-system} image for a Microsoft Azure cluster
+
+By default, the installation program downloads and installs the {op-system-first} image that is used to boot control plane and compute machines. With this enhancement, you can now override the default behavior by modifying the installation configuration file (`install-config.yaml`) to specify a custom {op-system} image. Before you deploy the cluster, you can modify the following installation parameters:
+
+* `compute.platorm.azure.osImage.publisher`
+* `compute.platorm.azure.osImage.offer`
+* `compute.platorm.azure.osImage.sku`
+* `compute.platorm.azure.osImage.version`
+* `controlPlane.platorm.azure.osImage.publisher`
+* `controlPlane.platorm.azure.osImage.offer`
+* `controlPlane.platorm.azure.osImage.sku`
+* `controlPlane.platorm.azure.osImage.version`
+* `platform.azure.defaultMachinePlatform.osImage.publisher`
+* `platform.azure.defaultMachinePlatform.osImage.offer`
+* `platform.azure.defaultMachinePlatform.osImage.sku`
+* `platform.azure.defaultMachinePlatform.osImage.version`
```

Comment:
missed new field introduced in 4.14 `azure.osImage.plan` under compute/controlPlane/defaultMachinePlatform.
```
$ ./openshift-install explain installconfig.platform.azure.defaultMachinePlatform.osImage.plan
KIND:     InstallConfig
VERSION:  v1

RESOURCE: <string>
  Valid Values: "WithPurchasePlan","NoPurchasePlan"
  Plan is the purchase plan of the image. If omitted, it defaults to "WithPurchasePlan".

```

Reply:
> missed new field introduced in 4.14 `azure.osImage.plan` under compute/controlPlane/defaultMachinePlatform.

@jinyunma When sizing this effort with @r4f4, we agreed that it was best to leave this parameter undocumented. [1]

[1] https://issues.redhat.com/browse/CORS-2503?focusedId=22420589&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-22420589

Reply:
thanks for the information, then the update LGTM.